services/blockchainService.ts

```diff
@@ -24,13 +22,30 @@ export interface GetAddressTransactionsParameters {
   maxTransactionsToReturn: number
 }
 
+interface InputOutput {
+  value: number
```

Comment:
is this value an integer in 'satoshis'? If not so, I think this would be better stored as a Decimal

Reply:
it is indeed, done